docs/health.md

```diff
@@ -0,0 +1,63 @@
+# Health
+
+Health command check health of the cluster and common configuration issues. Health command currently validates these things configurations (let us know if you would like to add other validation in health command):
+
+1. at least three mon pods should running on different nodes
+2. mon quorum and ceph health details
+3. at least three osd pods should running on different nodes
+4. all pods 'Running' status
+5. placement group status
+6. at least one mgr pod is running
+
+Health commands logs have three ways of logging:
+
+1. `Info`: This is just a logging information for the users.
+
+2. `Warning`: which mean there is some improvement required in the cluster.
+
+3. `Error`: This reuires immediate user attentions to get the cluster in healthy state.
+
+## Output
+
+```bash
+kubectl rook-ceph health
+
+# Info:  Checking if at least three mon pods are running on different nodes
+# Warning:  At least three mon pods should running on different nodes
+# rook-ceph-mon-a-5988949b9f-kfshx                1/1     Running       0          26s
+# rook-ceph-mon-a-debug-6bc9d99979-4q2hd          1/1     Terminating   0          32s
+# rook-ceph-mon-b-69c8cb6d85-vg6js                1/1     Running       0          2m29s
+# rook-ceph-mon-c-6f6754bff5-746rp                1/1     Running       0          2m18s
+
+# Info:  Checking mon quorum and ceph health details
+# Warning:  HEALTH_WARN 1/3 mons down, quorum b,c
+# [WRN] MON_DOWN: 1/3 mons down, quorum b,c
+#     mon.a (rank 0) addr [v2:10.98.95.196:3300/0,v1:10.98.95.196:6789/0] is down (out of quorum)
+
+# Info:  Checking if at least three osd pods are running on different nodes
+# Warning:  At least three osd pods should running on different nodes
```

Comment:
Since this is all part of the same output (I think) I think we might want to put `#` on the newlines as well.

```suggestion
# Info:  Checking if at least three mon pods are running on different nodes
# Warning:  At least three mon pods should running on different nodes
# rook-ceph-mon-a-5988949b9f-kfshx                1/1     Running       0          26s
# rook-ceph-mon-a-debug-6bc9d99979-4q2hd          1/1     Terminating   0          32s
# rook-ceph-mon-b-69c8cb6d85-vg6js                1/1     Running       0          2m29s
# rook-ceph-mon-c-6f6754bff5-746rp                1/1     Running       0          2m18s
#
# Info:  Checking mon quorum and ceph health details
# Warning:  HEALTH_WARN 1/3 mons down, quorum b,c
# [WRN] MON_DOWN: 1/3 mons down, quorum b,c
#     mon.a (rank 0) addr [v2:10.98.95.196:3300/0,v1:10.98.95.196:6789/0] is down (out of quorum)
#
# Info:  Checking if at least three osd pods are running on different nodes
# Warning:  At least three osd pods should running on different nodes
```

Reply:
done